client/src/app/component/bigqrcode/bigqrcode.html

```diff
@@ -85,8 +85,12 @@ <h1 uib-collapse="isCollapsed"><i class="glyphicon glyphicon-blackboard"></i> Mu
             </svg>
         </g>
     </svg>
+    <div class="row">
+        <br/>
```

Comment:
I know ;-) but in fact the big QRCODE is stock to the page footer, si I want to add a row space